src/classes/AFFL_ContactAccChange_TEST.cls

```diff
@@ -204,4 +204,210 @@ public with sharing class AFFL_ContactAccChange_TEST {
         TDTM_ProcessControl.resetRecursionFlag(TDTM_ProcessControl.registeredTrigger.AFFL_MultiRecordType_TDTM_Before_Insert);
         TDTM_ProcessControl.resetRecursionFlag(TDTM_ProcessControl.registeredTrigger.AFFL_MultiRecordType_TDTM_After_Update);
     }    
+
+    /* Tests changing the Account on a Primary Affiliation to an Account of the same Record Type when API names are used for 
+       'Contact Primary Affl Field ' in Affiliation Mappings will update the Primary Affiliation field on the Contact.
+    */
+
+    @isTest
+    private static void changeAccountSameTypeWithMappingsWithAPINamesForPrimaryAffl() {
+
+        AFFL_ContactAccChange_TEST.dataSetup();
+
+        // Update Affiliation mapping to use API names for Primary_Affl_Field__c
+        String namespace = UTIL_Namespace.getNamespace();
+        String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
+
+        for (Affl_Mappings__c mapping : UTIL_CustomSettingsFacade.getAfflMappings()) {
+            if (mapping.Primary_Affl_Field__c == 'Primary Business Organization') {
+                mapping.Primary_Affl_Field__c = prefix + 'Primary_Organization__c';
+            }
```

Comment:
Looks like querying the mappings and changing them doesn't update the mappings, just wanted to make sure if this is a right way to change the mappings @MatthewBlanski

Reply:
That seems accurate to me, given that `AFFL_MultiRecordTypeMapper.populateMaps()` should have been called as part of `AFFL_ContactAccChange_TEST.dataSetup()` when the affiliation was inserted, rendering those changes ineffective or at least incomplete. This does not seem like an appropriate way to handle that test, which is likely why our tests have been passing when they should not have been.

Please create a helper method in this test class that makes additional appropriate changes based on AFFL_MultiRecordTypeMapper.populateMaps()`. We'll be able to incorporate that into a service stub in the future.

Reply:
Updated per feedback. Resolving.

Reply:
Unresolving convo. I am not sure if I got it right @MatthewBlanski, can you please take a quick look tomorrow and let me know? I am now updating the custom setting mapping (instead of updating the values of the map in populateMaps() from our earlier convo as it gets over-written by the TDTM class by calling the populateMaps()). I am not certain if we still need to call afflMapper.populateMaps(); as it gets over-written by the TDTM again, so commented it for now. TIA.

Reply:
@MatthewBlanski Just wanted to make sure I addressed everything accordingly here.

Reply:
@MatthewBlanski Tagging you for your review. 👍